packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -683,6 +683,12 @@ const main = async (opts) => {
         })
     }
 
+    if (context.answers.project.commerce?.instanceUrl) {
```

Comment:
@vcua-mobify is this null check for a realistic scenario? Because if there's a chance of a falsy value here, then you'll be generating a template that looks like this here

```
{
                host: '',
                path: 'ocapi'
```

https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1260/files#diff-75bec535486c09cc39af03d91698eb1186ac0f76250e14175dd7dfcb5fc5796eR77

Reply:
This should never be a falsy value. In the options that require it (the template-retail-react-app options), we prompt the user to enter a valid URL for `project.commerce.instanceUrl`. `project.commerce` as a grouping of properties is also only not null for retail-react-app projects.

The CI tests were failing on other templates (such as express-minimal) where `project.commerce` is null and we were trying to access the instanceUrl property of it.

Reply:
The suggested change has been added